tests/unit/awaitStagingDeploysTest.ts

```diff
@@ -1,29 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
 /**
  * @jest-environment node
  */
 import * as core from '@actions/core';
-import _ from 'underscore';
+import asMutable from '@src/types/utils/AsMutable';
 import run from '../../.github/actions/javascript/awaitStagingDeploys/awaitStagingDeploys';
 import GithubUtils from '../../.github/libs/GithubUtils';
 
+type Workflow = {
+    workflow_id: string;
+    branch: string;
+};
+
+type WorkflowStatus = {status: string};
+
 // Lower poll rate to speed up tests
 const TEST_POLL_RATE = 1;
-const COMPLETED_WORKFLOW = {status: 'completed'};
-const INCOMPLETE_WORKFLOW = {status: 'in_progress'};
+const COMPLETED_WORKFLOW: WorkflowStatus = {status: 'completed'};
+const INCOMPLETE_WORKFLOW: WorkflowStatus = {status: 'in_progress'};
+
+type MockListResponse = {
+    data: {
+        workflow_runs: WorkflowStatus[];
+    };
+};
+
+type MockedFunctionListResponse = jest.MockedFunction<() => Promise<MockListResponse>>;
 
 const consoleSpy = jest.spyOn(console, 'log');
 const mockGetInput = jest.fn();
-const mockListPlatformDeploysForTag = jest.fn();
-const mockListPlatformDeploys = jest.fn();
-const mockListPreDeploys = jest.fn();
-const mockListWorkflowRuns = jest.fn().mockImplementation((args) => {
+const mockListPlatformDeploysForTag: MockedFunctionListResponse = jest.fn();
+const mockListPlatformDeploys: MockedFunctionListResponse = jest.fn();
+const mockListPreDeploys: MockedFunctionListResponse = jest.fn();
+const mockListWorkflowRuns = jest.fn().mockImplementation((args: Workflow) => {
     const defaultReturn = Promise.resolve({data: {workflow_runs: []}});
 
-    if (!_.has(args, 'workflow_id')) {
+    if (!args.workflow_id) {
```

Comment:
The logic is a bit different, this is more accurate:
```ts
    if (!('workflow_id' in args)) {
```

But the suggested change should work, right?